consensus/src/sync/syncer.rs

```diff
@@ -16,6 +16,7 @@ use nimiq_network_interface::network::{CloseReason, Network, NetworkEvent, Subsc
 use nimiq_primitives::policy::Policy;
 use nimiq_time::{interval, Interval};
 use nimiq_utils::stream::FuturesUnordered;
+use tokio::sync::broadcast::{channel as broadcast, Sender as BroadcastSender};
```

Comment:
```suggestion
use tokio::sync::broadcast;
```
And then use `broadcast::channel()` and `broadcast::Sender`. We use it this way in the rest of the code, but I don't know how to enforce that with a clippy lint…